contrib/macdeploy/README.md

```diff
@@ -15,8 +15,10 @@ When complete, it will have produced `Bitcoin-Core.dmg`.
 Our current macOS SDK
 (`Xcode-12.1-12A7403-extracted-SDK-with-libcxx-headers.tar.gz`) can be
 extracted from
-[Xcode_12.1.xip](https://download.developer.apple.com/Developer_Tools/Xcode_12.1/Xcode_12.1.xip).
+[Xcode_12.1.xip](https://download.developer.apple.com/Developer_Tools/Xcode_12.1/Xcode_12.1.xip)
+(alternatively, after login to your account go to "Downloads", then "More" and look for [`Xcode_12.1`](https://download.developer.apple.com/Developer_Tools/Xcode_12.1/Xcode_12.1.xip)).
```

Comment:
Might be good to also note cookies need to be enabled for the hostname.

Reply:
Done.
```diff
diff --git a/contrib/macdeploy/README.md b/contrib/macdeploy/README.md
index cdf2b6609f..c2cc7c58b3 100644
--- a/contrib/macdeploy/README.md
+++ b/contrib/macdeploy/README.md
@@ -16,10 +16,9 @@ Our current macOS SDK
 (`Xcode-12.1-12A7403-extracted-SDK-with-libcxx-headers.tar.gz`) can be
 extracted from
 [Xcode_12.1.xip](https://download.developer.apple.com/Developer_Tools/Xcode_12.1/Xcode_12.1.xip).
-Cookies must be enabled for this link to work after logging in.
 Alternatively, after logging in to your account go to 'Downloads', then 'More'
 and look for [`Xcode_12.1`](https://download.developer.apple.com/Developer_Tools/Xcode_12.1/Xcode_12.1.xip).
-An Apple ID is needed to download this.
+An Apple ID and cookies enabled for the hostname are needed to download this.
 The `sha256sum` of the archive should be `612443b1894b39368a596ea1607f30cbb0481ad44d5e29c75edb71a6d2cf050f`.
 
 After Xcode version 7.x, Apple started shipping the `Xcode.app` in a `.xip`
```